ipc/nodepart/arwenDriver.go

```diff
@@ -230,7 +260,8 @@ func (driver *ArwenDriver) IsClosed() bool {
 
 // RunSmartContractCreate sends a deploy request to Arwen and waits for the output
 func (driver *ArwenDriver) RunSmartContractCreate(input *vmcommon.ContractCreateInput) (*vmcommon.VMOutput, error) {
-	driver.nodeLogger.Trace("RunSmartContractCreate")
+	driver.counterDeploy++
```

Comment:
this might overflow.

Reply:
Only after Arwen processes 18,446,744,073,709,551,615 smart contracts without any restart :) With 1000 SC per second, this will be in ~ 584942417 years :)